ovos_core/skill_manager.py

```
@@ -160,13 +161,38 @@ def _handle_settings_file_change(self, path: str):
+    def _sync_gui_state(self):
+        """Refresh cached GUI availability without blocking skill scans."""
+        if self._gui_event.is_set():
+            return True
+
+        try:
+            response = self.bus.wait_for_response(
+                Message("gui.status.request"),
+                "gui.status.request.response",
+                timeout=self._status_query_timeout
+            )
+        except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)